src/main/java/homework/home08012019/MainLambdaTemplateStudents.java

```diff
@@ -0,0 +1,179 @@
+		int[] prices = fruits.stream().mapToInt(Fruit::getPrice).toArray();
+		System.out.println(Arrays.toString(prices));
+
+		System.out.println("\nConvert to mapToDouble Stream Price:");
+		double[] prices2 = fruits.stream().mapToDouble((Fruit::getPrice)).toArray();
+		System.out.println(Arrays.toString(prices2));
+
+		System.out.println("\nConvert to mapToLong Stream Price:");
+		long[] prices3 = fruits.stream().mapToLong((Fruit::getPrice)).toArray();
+		System.out.println(Arrays.toString(prices3));
+
+		System.out.println("\nShow vitamins for every fruit");
+		fruits.stream().peek(fr -> System.out.println("For " + fr.getFruitType() + " we show "))
+				.flatMap(fr -> fr.getVitamins().stream()).forEach(System.out::println);
+
+		Stream.of(asList("one","two"),asList("three", "four")).flatMap(x -> x.stream()) ;
+
+		System.out.println("\nShow all vitamins that present now in fruits:");
+		fruits.stream().flatMap(fr -> fr.getVitamins().stream()).distinct().forEach(System.out::println);
+
+
+		System.out.println("\nFind first in stream:");
+		System.out.println(fruits.stream().findFirst().get());
+
+		System.out.println("\nFindAny in stream:");
+		System.out.println(fruits.stream().findAny().get().getFruitType());
+
+		System.out.println("\nList of deliveryDates");
+		System.out.println(fruits.stream().map(Fruit::getDeliveryDate).collect(Collectors.toList()));
+
+		System.out.println("\nCount ORANGE");
+		System.out.println(fruits.stream().filter(fr -> ORANGE.equals(fr.getFruitType())).count());
+
+		System.out.println("\nHave we fruits that can live more 50 days, should be true because we have ORANGE?");
+		System.out.println(fruits.stream().anyMatch(fr -> fr.getDayToLive() > 50));
+
+		System.out.println("\nCannot any fruits live more 61 days?");
+		System.out.println(fruits.stream().noneMatch(fr -> fr.getDayToLive() > 61));
+
+		System.out.println("\nCan all fruits live more 5 days?");
+		System.out.println(fruits.stream().allMatch(fr -> fr.getDayToLive() >= 5));
+
+		System.out.println("\nMinimum time to live is:");
+		System.out.println(fruits.stream().map(Fruit::getDayToLive).min(Comparator.naturalOrder()).get());
+
+		System.out.println("\nMaximum time to live is:");
+		System.out.println(fruits.stream().map(Fruit::getDayToLive).max(Comparator.naturalOrder()).get());
+
+		System.out.println("\nShow forEach parallel:");
+		fruits.parallelStream().forEach(System.out::println);
+
+		System.out.println("\nShow forEachOrdered parallel:");
+		fruits.parallelStream().forEachOrdered(System.out::println);
+
+		System.out.println("\nShow array of deliveryDates:");
+		LocalDate[] arrayDate = fruits.stream().map(Fruit::getDeliveryDate).toArray(LocalDate[]::new);
+		System.out.println(Arrays.toString(arrayDate));
+
+		System.out.println("\nShow border timeToLive(bigger then) and price(less then):");
+		Fruit complexFruit = fruits.stream().reduce((fr1, fr2) -> {
+			Fruit result = new Fruit(null, -1, null, -1, null);
```

Comment:
You copy-paste my code and did some changes =)

Reply:
System.out.println(fruits.stream().map(Fruit::getPrice).reduce(0,Integer::sum));
this is my own code with command:  reduce!